examples/master.js

```diff
@@ -335,6 +335,10 @@ function onPeerConnectionFailed(remoteClientId, printLostConnectionLog = true, h
  * @param {boolean} formValues.sendSrflxCandidates - Send STUN candidates setting.
  * @param {boolean} formValues.sendRelayCandidates - Send TURN candidates setting.
  * @param {string} formValues.region - AWS region used to construct the STUN server URL.
+ * @param {boolean} formValues.turnWithUdp - If this particular TURN server should be used for the connection.
+ * @param {boolean} formValues.turnsWithUdp - If this particular TURN server should be used for the connection.
+ * @param {boolean} formValues.turnsWithTcp - If this particular TURN server should be used for the connection.
```

Comment:
It's good to have this option, but if it's not supported, then we may need to add an alert to indicate the same.

Reply:
Sounds good, we can add in a future PR.